app/src/main/res/values/component_colors.xml

```diff
@@ -40,4 +40,28 @@
   <color name="component_color_profile_and_device_id_activity_primary_text_color">@color/color_palette_primary_text_color</color>
   <color name="component_color_profile_and_device_id_activity_enabled_icon_button_color">@color/color_palette_primary_text_color</color>
   <color name="component_color_profile_and_device_id_activity_disabled_icon_button_color">@color/oppia_disabled_button_background</color>
+  <!-- Developer Options Activity -->
+  <color name="component_color_developer_options_activity_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_developer_options_activity_options_background_color">@color/color_palette_container_background_color</color>
+  <color name="component_color_developer_options_activity_sub_heading_color">@color/color_palette_primary_color</color>
+  <color name="component_color_developer_options_activity_menu_options_text_color">@color/color_palette_dark_text_color</color>
+  <color name="component_color_developer_options_activity_sub_heading_background_color">@color/color_palette_dark_background_color</color>
+  <color name="component_color_developer_options_activity_view_divider_color">@color/color_palette_dark_background_color</color>
+  <!-- View Events Logs Activity -->
+  <color name="component_color_view_events_logs_activity_background_color">@color/color_palette_dark_background_color</color>
```

Comment:
Please rename all colors for this activity to following
`***.....view_event_logs_activity_....`

Basically you need to replace `events` with `event`.